docs/cross-platform-development.md

```diff
@@ -0,0 +1,3 @@
+## Writing cross-platform apps
```

Comment:
I think "cross-platform-development.md" may be broader than we want it to be, but I don't have a better idea of how to make it more precise. Thoughts? What do we see this section evolving into?